queries/highlights.scm

```diff
@@ -8,6 +8,8 @@
   "operation"
   "list"
   "map"
+  "union"
+  "resource"
```

Comment:
Could you also add `"set"` here? Otherwise looks great!

Reply:
Uh, yes of course!